src/lib/useRoller.ts

```diff
@@ -225,22 +142,34 @@ export default function useRoller() {
         throw new Error('Internal Error: Missing Contracts/Web3/Wallet');
       }
 
+      if (Just.hasInstance(authMnemonic)) {
+        throw new Error(
+          "Auth key Error: A management mnemonic can't create invite codes"
+        );
+      }
```

Comment:
I'm capturing this here, but it could be done outside of the invite creation, for example when you click on the envelope—this depends on designs for when/where to show the error message, so tagging @jyng for feedback.

 It can be a more general question since I'm not sure what's the behavior for current Bridge when logging in with a management mnemonic and then trying to create some invites.